package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "web3-plugin-template",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

I think. We do not need to change the version so much with these changes. It would be better to set 1.0.1 for ex.

---

I also gree. 
I am not sure. But maybe something like:
```suggestion
  "version": "1.1.0",
```

---

good point, I'll check to make sure that the peer dependency update breaks anything before we can commit to a version update

---

FWIW I think 1.0.1 is a better choice for the version, per semantic versioning. Bumping the minor version (i.e. 1.1.0) would indicate that new non-breaking features were added. The third version number (patch) is the "fix" version and indicates that some existing feature was fixed. Ref: https://semver.org/